ms-ledger/routes/common_test.go

```diff
@@ -50,6 +53,15 @@ func getDefaultAccountLedgers() Accounts {
 func TestGetAllLedgers(t *testing.T) {
 	// Use community-recommended shorthand (known name clash)
 
+	mockAppService := &mocks.ApplicationService{}
+	mockAppService.On("AddRoute", mock.Anything, mock.Anything, mock.Anything, mock.Anything).
+		Return(nil)
```

Comment:
Where is this needed? If not needed (service API not called in this test) then service below can be set to nil and remove this. Same for other tests, i.e. only added when needed to keep the code as simple as possible.